MainDemo.Wpf/MainWindow.xaml

```diff
@@ -203,6 +203,7 @@
                             HorizontalAlignment="Center"
                             VerticalAlignment="Center"
                             FontSize="22"
+                            Margin="-152,0,0,0"
```

Comment:
Was it intentional to set a fixed negative margin to the main title text?

Reply:
Yes, We have 3 nav buttons and their actual size is 32. The first button margin from the left is 24 and for the other two buttons is 16. So:
 32 * 3 + (24 + 16 + 16) = 152